src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataAggregator.cs

```diff
@@ -64,14 +64,10 @@ public MetadataAggregator(MetadataReader baseReader, IReadOnlyList<MetadataReade
                 }
 
                 CalculateBaseCounts(baseReader, out baseTableRowCounts, out baseHeapSizes);
+                Debug.Assert(baseTableRowCounts != null);
```

Comment:
I took a look at the CalculateBaseCounts method and didn't see how it would return null, but I threw in the Debug.Assert for good measure.